docs/content/what-is-a-schedule-to-start-timeout.md

```diff
@@ -10,24 +10,35 @@ tags:
 import CenteredImage from "../components/CenteredImage.js"
 import RelatedReadList from '../components/RelatedReadList.js'
 
-A Schedule To Start Timeout is the maximum amount of time that is allowed, from when an [Activity Task](/docs/content/what-is-an-activity-task) is scheduled (placed in a Task Queue) to when a [Worker](/docs/content/what-is-a-worker) starts executing that Activity Task.
+A Schedule To Start Timeout is the maximum amount of time that is allowed, from when an [Activity Task](/docs/content/what-is-an-activity-task) is scheduled (placed in a Task Queue) to when a [Worker](/docs/content/what-is-a-worker) starts (picks it up from the Task Queue) that Activity Task.
+
+Thus, the moment that the Task is picked by the Worker is considered to be the start of the Activity Task for the Schedule To Start timeout and associated metrics.
+It's designed this way to avoid clock difference between the Server and the Worker.
```

Comment:
“It’s” contraction is too colloquial for this docs and often can find a way to be more specific about what “It” is